src/createUseState.lua

```diff
@@ -11,7 +11,7 @@ end
 local function createUseState(component)
 	local setValues = {}
 
-	return function(defaultValue)
+	return function<T>(defaultValue: T): (T, (newValue: T) -> ())
```

Comment:
The setter also allows ((T) -> T) -> ().

Reply:
Ah, that wasn't included in the README (https://github.com/Kampfkarren/roact-hooks#usestate)
Should I also update that in this PR?

Reply:
Ya sure